changelog/unreleased/pull-3590

```diff
@@ -0,0 +1,7 @@
+Enhancement: Document Safe Passwords and Clarify B2 application key information
```

Comment:
We usually don't add changelog entries for documentation changes. Please remove the changelog entry.